lib/split/services/time_based_conversions.rb

```diff
@@ -0,0 +1,18 @@
+module Split
+  module Services
+    module TimeBasedConversions
+
+      def self.within_conversion_time_frame?(user_id, experiment_id)
+
+        window_of_time_for_conversion = Split.configuration.experiments[experiment_id]["window_of_time_for_conversion"]
```

Comment:
Just a general note here: While it doesn't matter for our purposes because we'll always be using yaml files going forward, this will fail if experiments aren't preconfigured in yaml.